common/build.gradle

```diff
@@ -36,6 +36,9 @@ dependencies {
         exclude group: 'com.google.guava', module: 'listenablefuture'
     }
     compileOnly 'com.jayway.jsonpath:json-path:2.9.0'
+    compileOnly("com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}")
+    compileOnly("com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}")
+    compileOnly group: 'com.networknt' , name: 'json-schema-validator', version: '1.4.0'
```

Comment:
Does this 1.4.0 come from somewhere for consistency? otherwise why not use the latest 1.5.x version?